src/anamoly_detection/anomaly_engine.py

```diff
@@ -0,0 +1,76 @@
+from scapy.all import Packet
+from src.anamoly_detection.equation_parser import parse_equation
```

Comment:
Please convert all of the class methods in this file to use python naming convention instead of camel case, additionally to match our other class definitions.